packages/cli-lib/lang/en.lyaml

```diff
@@ -686,6 +686,7 @@ en:
                 describe: "Name of the secret to be updated"
             success:
               update: "The secret \"{{ secretName }}\" was updated in the HubSpot account: {{ accountId }}"
```

Comment:
We should be using the account name set during CLI auth rather than its numeric ID whenever possible.

Reply:
Maybe instead of "component" here we should say "serverless functions"? Component is a term that's specific to projects.

Reply:
Ahh gotcha yeah, i can change that

Reply:
@markhazlewood42 Looks like we're using account ID in a lot of places, probably would make sense to do that separately